blockchain/vm/interpreter.go

```diff
@@ -123,6 +123,23 @@ func NewEVMInterpreter(evm *EVM) *EVMInterpreter {
 		cfg.JumpTable = jt
 	}
 
+	// Enable the opcode computation cost limit
+	if cfg.ComputationCostLimit == 0 {
+		switch {
+		case evm.chainRules.IsCancun:
+			// approximately 120 ms
+			cfg.ComputationCostLimit = uint64(params.OpcodeComputationCostLimitCancun)
+		default:
+			// approximately 100 ms
```

Comment:
```suggestion
```

Reply:
removed the line at https://github.com/klaytn/klaytn/pull/2037/commits/8d9deb0ecb40ec89244fd97a5b2c3800560685fe